tests/unit/test_coreneuron_directmode.py

```diff
@@ -194,3 +193,46 @@ def test_coreneuron_keep_build(create_tmp_simulation_config_file):
     assert (Path(SimConfig.output_root) / "sim.conf").exists()
     assert (Path(SimConfig.output_root) / "report.conf").exists()
     assert (Path(SimConfig.output_root) / "populations_offset.dat").exists()
+
+
+@pytest.mark.parametrize(
+    "create_tmp_simulation_config_file",
+    [
+        {
+            "simconfig_fixture": "ringtest_baseconfig",
+            "extra_config": {
+                "target_simulator": "CORENEURON",
+                "inputs": {
+                    "pulse": {
+                        "module": "pulse",
+                        "input_type": "current_clamp",
+                        "delay": 5,
+                        "duration": 50,
+                        "node_set": "RingA",
+                        "represents_physical_electrode": True,
+                        "amp_start": 10,
+                        "width": 1,
+                        "frequency": 50,
+                    }
+                }
+            }
+        }
+    ],
+    indirect=True,
+)
+def test_coreneuron_multicycle(create_tmp_simulation_config_file):
```

Comment:
Is this test already in test_coreneuron_multicycle.py ?

Reply:
obsolete code, removed. Thx!